### PR TITLE
index.md: Fix gradient around quotes

### PR DESCRIPTION
**Pull request by hemangsk**
Gradient is dark in center and becomes lighter
towards left and towards right.

### CODE REVIEW

**Comment by sils** [comment]
screenshot>

**Comment by sils** [comment]
?

**Comment by hemangsk** [comment]
screen shot 2017-01-01 at 10 58 32 pm


screen shot 2017-01-01 at 10 58 20 pm

**Comment by sils** [comment]
conflicts

File: index.md [hunk 1]
```diff
@@ -69,9 +69,11 @@ order: 1
   <div class="author-quote">Jozef Képesi, CTO at Skypicker</div>
 </div>
 
-<hr class="mid-sepr"><br>
-<div class="row">
-  <div class="col-md-6 col-xs-12 text-center">
+
+<hr class="mid-sepr"><br><br>
+<div class="row service-row">
```
Comment (sils) [inline index.md]: what's the service row for? @chrisnew removed it earlier

File: index.md [hunk 2]
```diff
@@ -69,9 +69,11 @@ order: 1
...
+  <div class="col-md-6">
+    <center>
```
Comment (sils) [inline index.md]: why're you adding another unclosed center tag

**Comment by hemangsk** [comment]
Images in center
screen shot 2017-01-02 at 2 38 48 pm

screen shot 2017-01-02 at 2 38 20 pm

screen shot 2017-01-02 at 2 43 30 pm

**Comment by sils** [comment]
ack 
db3093a

**Pull request merged**